ui/components/ui/hardware-wallet-state/hardware-wallet-state.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import HardwareWalletState from '.';
+
+export default {
+  title: 'Components/App/HardwareWalletState',
+
+  argTypes: {
+    pollingRateMs: {
+      control: 'number',
+    },
+    initialStatus: {
+      control: 'text',
```

Comment:
Could improve the use of this control. Not that I think it adds a huge amount of value in storybook form but can't hurt.

```suggestion
      control: 'select',
      options: ['locked', 'unlocked'],
```